docs/user-guide/example-config.md

```diff
@@ -132,6 +132,7 @@ You might want to learn a little about [how rules are named and how they work to
     "selector-list-comma-space-after": "always"|"never"|"always-single-line"|"never-single-line",
     "selector-list-comma-space-before": "always"|"never"|"always-single-line"|"never-single-line",
     "selector-max-empty-lines": int,
+    "selector-max-class": int,
```

Comment:
Sorry I'm chiming in late in the game here ... I was going to review this code and then merge, but had at last one or two notes. First one: Should we rename this to `selector-max-classes` (plural)? That seems to fit the convention better.